### ISSUE TITLE
Notation used in the spec

### ISSUE THREAD

**Issue by dpwiz** [ocapn/ocapn#68]
It looks like Preserves text form, but.. not quite.

Some ops are wrapping record args in `[]`, but the test suite (which apparently works, as seen on the recent call demo) uses record args directly.

```scheme
<op:bootstrap [answer-position    ; positive integer or 0
               resolve-me-desc]>  ; desc:import-object | desc:import-promise
```
vs
```python
        assert record.label == syrup.Symbol("op:bootstrap")
        assert len(record.args) == 2
        resolve_me_desc = decode_captp_message(record.args[1])
        return cls(record.args[0], resolve_me_desc)
```

So, those square brackets are misleading - either they are a single-sequence argument (they aren't) or they are "optional" args (they aren't either!).

**Comment by tsyesika** [comment]
Yeah, my bad, you're correct. It's a label with `op:bootstrap` and two arguments, not what I said. I think the spec has consistently expressed arguments to labels in this way, but it obviously isn't great to use this undefined syntax which apparently even confuses me at first glance.

**Comment by dpwiz** [comment]
Another thing I've found while making the [] changes is the kinda-record syntax is used as a type in a place, where should be a argument name instead:

```
<op:pick  [<promise-pos>         ; <desc:answer | desc:import-promise>
           <selected-value-pos>  ; Positive Integer
           <new-answer-pos>]>    ; Positive Integer
```

Perhaps that should be just argument and type names:
```
<op:pick promise-pos         ; Either desc:answer or desc:import-promise
         selected-value-pos  ; Positive Integer
         new-answer-pos>     ; Positive Integer
```

**Comment by dpwiz** [comment]
I just noticed there are actually two styles of code:
- Signatures: names with their types in comments.
- Examples: unfolded/inlined values with descriptions in comments.

:thinking:
